packages/server/lib/makeDataContext.ts

```diff
@@ -46,9 +47,20 @@ export function makeDataContext (options: MakeDataContextOptions): DataContext {
       close: browsers.close,
       getBrowsers,
       async ensureAndGetByNameOrPath (nameOrPath: string) {
+        const projectConfig: FullConfig = await (ctx as DataContext).project.getConfig()
```

Comment:
This type assertion seems like it should be unnecessary (and isn't used below) but I couldn't avoid the `any` without it. Improvements welcome!

Reply:
Hi 👋 

You can probably do `const projectConfig = await getCtx().project.getConfig()` to get the correct type inference (which will return a reference to `ctx`).